src/inmanta/app.py

```diff
@@ -964,7 +964,17 @@ def default_logging_config(options: argparse.Namespace) -> None:
     context = {var: f"{place_holder}{var}{place_holder}" for var in context_vars}
 
     component_config = InmantaLoggerConfig(stream=sys.stdout, no_install=True)
-    component_config.apply_options(options, options.cmd, context)
+    # Force TTY so that this command outputs the same config when piping to a file
```

Comment:
The call to `start_loggers_for_extensions()` also extends the logging config, so we need to make sure that the environment variable is also set for that call. Why not set is for this entire method?

Reply:
I'm not sure it uses the `_is_on_tty` which, I believe, is the only real relevant reason why we set the environment variable. I set the environment variable to the whole method anyway, just to avoid some errors in the future if we end up using the `_is_on_tty` in other parts of this code

Reply:
I know. My main concern here was to make it future proof.